### ISSUE TITLE
ANT and ANT+ on NRF52 boards

### ISSUE THREAD

**Issue by casainho** [adafruit/circuitpython#7310]
I wish to be able to use ANT and ANT+ on NRF52 boards. I use it for implementing ANT+LEV EBike standard as also GPS cycling display Garmin Edge page change but in C, similar to Nordic SDK code, if anyone is interested: https://github.com/OpenSourceEBike/EV_Display_Bluetooth_Ant/tree/main/firmware/display/antplus_lev

**Comment by dhalbert** [comment]
We have not supported ANT and ANT+ because of the licensing fee requirements:
https://www.thisisant.com/developer/components/view-all-components/nrf52-ant-stack-licensing-faq

When I was researching this a while ago, I found that some bike devices that support ANT+ also support BLE, such as the Wahoo sensors.

**Comment by dhalbert** [comment]
It is not completely clear to me if Adafruit would be required to pay the licensing fee, because we are selling boards that could load the ANT+-capable software, even if just for hobbyist use, or if the licensing fee is meant to apply only to commercial products that use ANT+.

**Comment by casainho** [comment]
Yes, the DIY EBike display I built have simultaneous Bluetooth and ANT+, it works very well: https://opensourceebike.github.io

But I would prefer to develop again that display based in Python code instead. And ANT+ is a MUST for cycling devices, from electronics gears shifters, tire pressure sensors and the EBike motors.

![image](https://user-images.githubusercontent.com/147128/206056357-c9d82cf5-bdb6-4793-8e04-ecb589b3f018.png)

![image](https://user-images.githubusercontent.com/147128/206056470-082c6212-8e34-4f03-97b4-158d95788405.png)

![image](https://user-images.githubusercontent.com/147128/206056484-4b97b182-cedd-45fb-9835-ebcc902fbf8f.png)

**Comment by cujomalainey** [comment]
FYI @orrmany wrote this awesome article on how to load some adafruit devices https://blogarak.wordpress.com/2020/03/15/s340-softdevice-adafruit-nrf52840-express-feather/
